pages/register.js

```diff
@@ -0,0 +1,40 @@
+const { I } = inject();
+
+module.exports = {
+  registerHeaderText: 'Register Account',
+  accountCreatedText: 'Your account Has Been Created!',
+  firstNameField: { css: '#input-firstname' },
+  lastNameField: { css: '#input-lastname' },
+  emailField: { css: '#input-email' },
+  telephoneField: { css: '#input-telephone' },
+  passwordField: { css: '#input-password' },
+  passwordConfirmField: { css: '#input-confirm' },
+  privacyPolicyButton: { xpath: '//*[@id="content"]/form/div/div/input[1]' },
+  continueButton: { xpath: '//*[@id="content"]/form/div/div/input[2]' },
+
+
+  verifyRegisterAccountText() {
+    I.see(this.registerHeaderText);
+
+
+  },
+
+  fillRegistrationDetails(user) {
+    I.fillField(this.firstNameField, user.firstName),
```

Comment:
Instruction usually ends with semicolon ';', not comma ','.
Comma is to separate object key values, array values, etc.